src/main/java/org/sagebionetworks/web/client/widget/table/modal/fileview/CreateTableViewWizardStep2.java

```diff
@@ -118,30 +123,45 @@ public void configure(Table entity, TableType tableType) {
     this.entity = entity;
     this.tableType = tableType;
 
-    editor.configure(tableType, new ArrayList<ColumnModel>());
+    List<ColumnModel> defaultColumns = getDefaultColumns();
 
-    boolean isView = !TableType.table.equals(tableType);
-    this.editor.setAddDefaultColumnsButtonVisible(isView);
-    this.editor.setAddAnnotationColumnsButtonVisible(isView);
-    if (isView) {
-      // start with the default file columns
-      getDefaultColumnsForView();
+    if (DisplayUtils.isInTestWebsite(this.cookies)) {
+      editorV2.configure(
+        EntityTypeUtils.getEntityType(entity),
+        getViewScope(),
+        defaultColumns
+      );
+    } else {
+      editor.configure(tableType, new ArrayList<>());
+      boolean isView = !TableType.table.equals(tableType);
+      this.editor.setAddDefaultColumnsButtonVisible(isView);
+      this.editor.setAddAnnotationColumnsButtonVisible(isView);
+      if (isView) {
+        // start with the default file columns
+        editor.addColumns(defaultColumns);
+      }
     }
   }
 
-  public void getDefaultColumnsForView() {
-    List<ColumnModel> defaultColumns = fileViewDefaultColumns.getDefaultViewColumns(
-      tableType
-    );
-    editor.addColumns(defaultColumns);
+  public void onAddDefaultViewColumnsCallback() {
+    editor.addColumns(getDefaultColumns());
   }
 
-  public void getPossibleColumnModelsForViewScope(String nextPageToken) {
-    view.setJobTrackerVisible(true);
-    presenter.clearErrors();
+  public List<ColumnModel> getDefaultColumns() {
+    boolean isView = !TableType.table.equals(tableType);
+    if (isView) {
+      return fileViewDefaultColumns.getDefaultViewColumns(tableType);
+    } else {
+      return Collections.emptyList();
+    }
+  }
+
+  public ViewScope getViewScope() {
     ViewScope scope = new ViewScope();
     List<String> scopeIds = null;
-    if (entity instanceof EntityView) {
+    if (entity instanceof TableEntity) {
+      return new ViewScope();
```

Comment:
A TableEntity has a ViewScope?

Reply:
I think I hit an NPE but the right thing to do would be to return null and explicitly handle the null case, thanks for catching this.